substrate/primitives/consensus/grandpa/src/lib.rs

```diff
@@ -469,55 +462,6 @@ pub const PENDING_CHANGE_CALL: &str = "grandpa_pending_change";
 /// WASM function call to get current GRANDPA authorities.
 pub const AUTHORITIES_CALL: &str = "grandpa_authorities";
```

Comment:
```suggestion
```

Unrelated to this PR but these can be removed (they're unused). The one above as well but github doesn't allow me to comment on it.